src/event-handlers/court-emails.js

```diff
@@ -1,4 +1,4 @@
-const qs = require('quertystring');
+const qs = require('querystring');
```

Comment:
Ha! this one must've been annoying

Reply:
Yeah, it looks like my brain is kind of hard-wired to type `querty` whenever I want to type `query`. 😅